### DISCUSSION TITLE
Enable debug log for plugin only

### DISCUSSION THREAD

**Discussion by ihatetoregister** [jellyfin/jellyfin#9546] [Q&A] [answered]
Is it possible to enable debug traces only for a plugin?

MinimumLevel sets logging for the whole application but I'm after something more fine grained. 

I've tried using the category but it doesn't seem to work:

```
private readonly ILogger<ServerEntryPoint> _logger;
```

```
{
  "Serilog": {
    "ServerEntryPoint": "Debug"
  }
}
```

**Comment by crobibero** [comment, marked as answer]
Yes, see here for an example

https://github.com/jellyfin/jellyfin-plugin-webhook#my-webhook-isnt-working

**Comment by ihatetoregister** [comment]
That did the trick. Here is my logger.json for completion

```
{
  "Serilog": {
      "MinimumLevel": {
          "Default": "Information",
          "Override": {
              "Microsoft": "Warning",
              "System": "Warning",
              "Jellyfin.Plugin.Lastfm": "Debug"
          }
      }
  }
}
```
